bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -845,6 +854,30 @@ protected void release(boolean wait)
       }
     }
 
+    private boolean canEvict(boolean writer)
+    {
+      if (backPressureEnabled) {
+        if (!writer) {
+          boolean evict = true;
+          int blocks = 0;
+          // Search backwards from current block as opposed to searching forward from first block till current block as
+          // it is more likely to find the match quicker. No need to search more than maximum number of in memory permits.
+          for (Block temp = this.prev; (blocks < (MAX_COUNT_OF_INMEM_BLOCKS - 1)) && (temp != null); temp = temp.prev, ++blocks) {
+            if (temp.refCount.intValue() != 0) {
```

Comment:
No, I prefer get() to be consistent with the rest of the DataList code and to indicate that there is no conversion is done. intValue() calls get(), why to have extra call, even if hotspot will optimize it eventually?